doc/classes/EditorSettings.xml

```diff
@@ -1025,6 +1141,9 @@
 		<member name="text_editor/behavior/files/convert_indent_on_save" type="bool" setter="" getter="">
 			If [code]true[/code], converts indentation to match the script editor's indentation settings when saving a script. See also [member text_editor/behavior/indent/type].
 		</member>
+		<member name="text_editor/behavior/files/open_dominant_script_on_scene_change" type="bool" setter="" getter="">
+			If [code]true[/code], opening a scene will automatically open its main script, i.e. the script attached to the root node, or the topmost node if the root has no script.
```

Comment:
I don't think the concept of "main" or "dominant" script appears anywhere else in the documentation but I may be mistaken. If that's true though, there's no need to give it a name and then a definition.
```suggestion
			If [code]true[/code], opening a scene automatically opens the script attached to the root node, or the topmost node if the root has no script.
```